nat-lab/tests/utils/connection_util.py

```diff
@@ -368,3 +375,20 @@ def generate_connection_tracker_config(
             ),
         ),
     ]
+
+    # Add IPv6 configs
+    if connection_tag in LAN_ADDR_MAP_V6:
+        ctc_list.append(
+            ConnectionTrackerConfig(
+                "stun6",
+                stun6_limits,
+                FiveTuple(
+                    protocol="udp",
+                    src_ip=LAN_ADDR_MAP_V6[connection_tag],
+                    dst_ip=config.STUNV6_SERVER,
+                    dst_port=3478,
```

Comment:
We do not serve STUNv6 on dst_port 3478 anywhere currently.

Reply:
We'll do have a separate ipv6 stun server, after this PR will be merged.